pkg/features/versioned_kube_features.go

```diff
@@ -39,6 +39,10 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 		{Version: version.MustParse("1.31"), Default: false, PreRelease: featuregate.Deprecated},
 	},
 
+	AllowResourceResizeInContainerWithExclusiveCPUs: {
+		{Version: version.MustParse("1.32"), Default: false, PreRelease: featuregate.Deprecated},
```

Comment:
Thought Deprecated , because it will be incomplete ie when set to true it will be silently ignored since the functionality will not be there yet until fix is merged. I don’t have a strong opinion on that, I am ok with alpha as well if that is acceptable.

Reply:
Sorry missed comment in Slack, thanks _"This shouldn't be deprecated, since we're not removing the functionality. We're just holding back a subset of the functionality in alpha."._ Done